### ISSUE TITLE
Largo Explore Related Widget breaks Slideshows in posts with categories.

### ISSUE THREAD

**Issue by benlk** [INN/Largo#1240]
Javascript console in the browser fails thusly:

    Uncaught Error: Syntax error, unrecognized expression: a[href^=#]

Where does Largo have that?

```
js/jquery.idTabs.js:110: 
js/jquery.idTabs.js:212: 
js/jquery.idTabs.js:213: 
```

Does the affected site's child theme have that syntax? **No**.

Does removing the category and tags prevent that from appearing? **Yes.**

Does it happen only with tags? That error is not there with tags.

Does it happen only with categories? With categories but no tags, it happens.

What's the difference between those two pages? The old-style largo-explore-related-widget loads when a post has categories. That widget uses tabs.

**Comment by benlk** [comment]
List of affected site IDs is at the bottom of this list: https://github.com/INN/wp-scripts/issues/5

- [x] Temporary-temporary solution: Replace that widget on the affected sites.

**Comment by aschweigert** [comment]
see #1241

**Comment by benlk** [comment]
Adding this here to make it more findable: `#rp0` `#rp1`

And this will be closed with 0.5.5.

**Comment by benlk** [comment]
#1241 merged; I'm closing this.
